src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx

```diff
@@ -77,7 +77,7 @@ Thresholds are set during the process of [creating a condition](/docs/alerts/new
   </tbody>
 </table>
 
-To learn more about specific alert condition types (like baseline and NRQL), see [Types of conditions](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#condition-types).
+To learn more about specific alert condition types (like anomaly and NRQL), see [Types of conditions](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#condition-types)..
```

Comment:
Looks like there's a double period at the end of this sentence

Reply:
fixed